util.tsx

```diff
@@ -0,0 +1,15 @@
+import useSWR from "swr";
```

Comment:
Does this file need to be a `tsx`? As long it doesn't return any JSX content, it is better to name it as `util.ts`. Also, it might be better to have something more specific than a generic name like  `util.ts`.

Reply:
Sure I can rename the extension. I'll also rename the filename to `api_util.ts`

Reply:
Thanks @ethansiegl. Can we use camel case (`apiUtil.ts`) or title case (`ApiUtil.ts`) instead? I prefer title case in general, but of course there is no universal rule for file naming. We shouldn't use title case for files under `pages` though, because they're mapped to actual routes.